NuCivic/dkan
# [CIVIC-6101] Adapt the 'dataQuality' input value druing POD harvest

Issue: CIVIC-6101

## Description
As per the [pod specification](https://project-open-data.cio.gov/v1.1/schema/#dataQuality), `dataQuality` should be a boolean value. The `field_odfe_data_quality` destination field is expecting strings as input. We need to convert those boolean values to strings.  

## Acceptance Criteria
1. `dataQuality` POD field should be imported correctly.

## QA steps
1. Create a harvest source. The source datasets should have the optional `dataQuality` field.
2. Harvest the source.
3. **Expected**: The `field_odfe_data_quality` (data Quality) should show a boolean value (True or False) on the dataset view page.
**Actual**: The `field_odfe_data_quality` have "1" or "0" value.


## Merge process
None needed.

## Reminders
